lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -1291,6 +1291,39 @@ defmodule Kernel.SpecialForms do
   defmacro for(args)
 
   @doc """
+  Used to combine matching clauses.
+
+  Let's start with an example:
+
+      iex> opts = %{width: 10, height: 15}
+      iex> with {:ok, width} <- Map.fetch(opts, :width),
+      ...>      {:ok, height} <- Map.fetch(opts, :height),
+      ...>   do: {:ok, width * height}
+      {:ok, 150}
+
+  If all clauses match, the `do` block is executed, returning its result.
+  Otherwise the chain is aborted and a non-matched value is returned:
```

Comment:
wouldn't it be clearer to say `Otherwise the chain is aborted and `:error` is returned:`
or is there any other non-matched value?